### PR TITLE
Fix/boas 834

### PR DESCRIPTION
**Pull request by mayya-bond-kc**

### CODE REVIEW

File: app/components/back-office-app-services/document-check-function/function-app.tf [hunk 1]
```diff
@@ -17,10 +17,10 @@ module "anti_virus_functions" {
...
 
   app_settings = {
-    CLAM_AV_HOST              = var.clamav_host
-    CLAM_AV_PORT              = "3310"
-    API_HOST                  = var.back_office_api_host
-    DOCUMENT_STORAGE_API_HOST = var.back_office_document_api_host
+    CLAM_AV_HOST                          = var.clamav_host
+    CLAM_AV_PORT                          = "3310"
+    API_HOST                              = var.back_office_api_host
+    pinsstdocsbodevukw001__blobServiceUri = var.back_office_document_api_host
```
Comment (chrisdalykinandcarta) [inline app/components/back-office-app-services/document-check-function/function-app.tf:23]: We don't need this any more, did you rebase from main?

Comment (mayya-bond-kc) [reply app/components/back-office-app-services/document-check-function/function-app.tf:23]: this was an old change that was from before you made your changes, let me remove it

File: app/components/back-office-app-services/document-check-function/function-app.tf [hunk 2]
```diff
@@ -17,10 +17,9 @@ module "anti_virus_functions" {
   use_app_insights                         = true
 
   app_settings = {
-    CLAM_AV_HOST              = var.clamav_host
-    CLAM_AV_PORT              = "3310"
-    API_HOST                  = var.back_office_api_host
-    DOCUMENT_STORAGE_API_HOST = var.back_office_document_api_host
```
Comment (chrisdalykinandcarta) [inline app/components/back-office-app-services/document-check-function/function-app.tf:23]: We still need this too

Comment (mayya-bond-kc) [reply app/components/back-office-app-services/document-check-function/function-app.tf:23]: nice spot

**Pull request merged**